Clover/app/src/main/java/org/floens/chan/core/manager/WatchManager.java

```diff
@@ -224,6 +224,23 @@ public void deletePin(Pin pin) {
         EventBus.getDefault().post(new PinRemovedMessage(pin));
     }
 
+    public void deletePins(List<Pin> pinList) {
+        for (Pin pin : pinList) {
+            pins.remove(pin);
+            destroyPinWatcher(pin);
+        }
+
+        databaseManager.runTask(databasePinManager.deletePins(pinList));
+
+        // Update the new orders
+        sortListAndApplyOrders();
+        updatePinsInDatabase();
+
+        updateState();
+
+        EventBus.getDefault().post(new AllPinsRemovedMessage());
```

Comment:
This event bus message is not correct unless pinList is equivalent to WatchManager's pins variable. This will cause the error noted here: https://github.com/Adamantcheese/Clover/issues/2.